appauthlibrary/build.gradle

```diff
@@ -0,0 +1,32 @@
+apply plugin: 'com.android.library'
+apply from: "${rootDir}/gradle/android-common.gradle"
+version = "${rootProject.versionName}"
+
+android {
+
+    defaultConfig {
+        project.archivesBaseName = "appauth-android"
+        manifestPlaceholders = [
+                'appAuthRedirectScheme': ''
```

Comment:
This scheme is going to be replaced by the value provided in manifestPlaceholders of the client's app during manifest merging stage. But we can not remove it because we need this for running our tests.

Reply:
I'm not following, can you explain this?

Reply:
In the first implementation I have just added app-auth as a sourse set, so we can have app-auth sources provided as a part of our library.

But we have found problem (I think I wrote about it in #mobile-sdk) that we forgot to add *manifest.xml* from app-auth that broke that we need because otherwise  *RedirectUriReceiverActivity* will not be registered within android Activity Manager and will not available.

In order to solve this problem with manifest I have decided to create a separate module  *appauthlibrary* that is responsible for adding only app-auth and manifest and than I add it as a dependency to our library in order to add *RedirectUriReceiverActivity* to our manifest

Reply:
Can add comment to the build.gradle or a README to appauthlibrary/ ?

Reply:
I will add a comment to build.gradle